### ISSUE TITLE
Not displaying dumps

### ISSUE THREAD

**Issue by slovenianGooner** [laravel/telescope#135]
Anything special that we have to do to display dumps? It gets caught in the code but does not display on Telescope.

**Comment by fridzema** [comment]
Dump works perfectly for me.
Have you checked your environment and checks?

`app/Providers/TelescopeServiceProvider.php`
By default it is only capturing in local environment.

Telescope::filter(function (IncomingEntry $entry) {
    if ($this->app->environment() == 'local') {
        return true;
    }

    return $entry->isReportableException() ||
           $entry->isFailedJob() ||
           $entry->isScheduledTask() ||
           $entry->hasMonitoredTag();
});

**Comment by slovenianGooner** [comment]
Yeah it's a local environment - it seems to "swallow" the dump in the response as in it's not visible but it doesn't show in the dumps tab.

If I move away from the dumps tab, the dump is displayed in the response.

I have not modified anything in the Service Provider - so find it really strange.

**Comment by RicardoRamirezR** [comment]
I've changed it to:
```php
if ($this->app->environment() != 'production') {
    return true;
}
```

**Comment by JPGrefaldo** [comment]
When I try to dump($var), the vardumper(i.e Symfony\Component\VarDumper\VarDumper) and display the dump but not in Telescope. I tried your suggestions, still no luck.

**Comment by TinoN** [comment]
> I found the issue in vendor/beyondcode/laravel-dump-server/config/config.php
> 
> The host had a port number 9912 assigned to the host. I removed it, and it worked

I changed the port to :8000, now it's working
